src/one_dragon/base/push/channel/work_weixin.py

```
@@ -0,0 +1,201 @@
+            text_data = {"msgtype": "text", "text": {"content": f"{title}\n{content}"}}
+            try:
+                resp_obj = requests.post(url, data=json.dumps(text_data), headers=headers, timeout=15)
+                resp_obj.raise_for_status()
+
+                status = resp_obj.status_code
+                body_snip = (resp_obj.text or "")[:300] if hasattr(resp_obj, "text") else ""
+                resp = None
+
+                try:
+                    resp = resp_obj.json()
+                except Exception as je:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)